### PR TITLE
v0.1.12: Support for backpack plugin integration & Fix several bugs

### LINKED ISSUE #37: Add Backpack plugin support
With the next iteration of the block workspace we may switch to the Blockly-provided backpack plugin. If multiple blocks are selected, we probably want our own menu substitution that copies the N selected blocks to the backpack.

### LINKED ISSUE #40: Unexpected focus is claimed on mouse enter of the workspace div
The following code claims focus when the mouse enters the workspace div:
```
 injectionDiv.addEventListener('mouseenter', () => {
      if (document.activeElement === this.workspace_.svgGroup_.parentElement ||
          document.activeElement.nodeName.toLowerCase() === 'input' ||
          document.activeElement.nodeName.toLowerCase() === 'textarea') {
        return;
      }
      this.workspace_.svgGroup_.parentElement.focus();
    });
```
As shown some exceptions are made on 'input' and 'textarea' however in my case another node type that is closed due to the change of focus. 

What functionality will break when this code is removed? It is probably not a good idea to extend the list of exceptions...

As shown in the video below a vscode menu is closed as the focus is set to the workspace div when I enter the div. 

https://github.com/mit-cml/workspace-multiselect/assets/8227977/9c3eb76d-95e5-4cd6-99f9-85966a81ff2e

### LINKED ISSUE #42: this.draggedConnectionManager_ is undefined
Hello,
when installing several plugins in my application, unless I've made some declaration errors, I've noticed that there are conflicts due to function names. With the @mit-app-inventor/blockly-plugin-workspace-multiselect plugin, which has the same 'copy' function name as the keyboard navigation plugin, I need to initialize the keyboard navigation plugin after the multi-select plugin, thanks ([https://groups.google.com/g/blockly/c/MuL2Ln8SwDU](https://groups.google.com/g/blockly/c/MuL2Ln8SwDU)) .

But if I select (keyboard shortcut or icon lick on workspace) it works and when I release by clicking group icon on workspace, here is the message:
this.draggedConnectionManager_ is undefined patchUpdateBlockAfterMove/this.BaseBlockDragger.prototype.updateBlockAfterMove_@webpack-internal:///./src/index.ts:9415:13 endDrag@webpack-internal:///./node_modules/blockly/blockly_compressed.js:1263:266 endDrag/<@webpack-internal:///./src/index.ts:9384:21 endDrag@webpack-internal:///./src/index.ts:9378:25 handleUp@webpack-internal:///./node_modules/blockly/blockly_compressed.js:667:212 f@webpack-internal:///./node_modules/blockly/blockly_compressed.js:84:159 EventListener.handleEvent*conditionalBind$$module$build$src$core$browser_events@webpack-internal:///./node_modules/blockly/blockly_compressed.js:84:393 bindMouseEvents@webpack-internal:///./node_modules/blockly/blockly_compressed.js:665:23 doStart@webpack-internal:///./node_modules/blockly/blockly_compressed.js:664:151 handleWsStart@webpack-internal:///./node_modules/blockly/blockly_compressed.js:673:8 wrappedFunc@webpack-internal:///./src/index.ts:9132:14 onMouseDown_@webpack-internal:///./node_modules/blockly/blockly_compressed.js:1089:428 f@webpack-internal:///./node_modules/blockly/blockly_compressed.js:84:149 

Thanks for your help.

### PR DESCRIPTION
**Pull request by HollowMan6**
- Fix the context menu error when integrating with backpack plugin
Right-click on a block in the backpack causes the following errors:

```bash
Cannot read properties of undefined (reading 'size') TypeError: Cannot read properties of undefined (reading 'size')
    at Object.preconditionFn (webpack-internal:///./src/multiselect_contextmenu.js:67:27)
......
```
- Add support for multi-selection of Copy to backpack context menu (as indicated in the README changes)
  Resolves #37 
- Never show as 'Delete 0 Blocks'
- Fix #40 
- Fix #42

### CODE REVIEW

**Comment by HollowMan6** [comment]
WX20240116-201546@2x

**Comment by HollowMan6** [comment]
Let's wait for the adaptation to Blockly 11 to be completed (#39) before we land this PR.

**Comment by ewpatton** [comment]
@HollowMan6 Sure thing.

**Comment by HollowMan6** [comment]
@ewpatton Let's merge this and release a new version (v0.1.12). This can be our last stable version for Blockly v10.

I suddenly became aware that we will have significant breaking changes for Blockly v11 (#39), so I guess it will be better to raise the minor version number to v0.2.x or the major version number to v1.0.x since I can't guarantee that it will still be as stable as the old ones.

~~> Let's wait for the adaptation to Blockly 11 to be completed (#39) before we land this PR.~~

I will then open a separate PR for this.

File: package.json [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
   "name": "@mit-app-inventor/blockly-plugin-workspace-multiselect",
-  "version": "0.1.11",
+  "version": "0.1.12",
   "description": "A Blockly plugin that allows to drag, select and doing actions on multiple blocks in the workspace.",
   "scripts": {
     "audit:fix": "blockly-scripts auditFix",
     "build": "blockly-scripts build",
     "clean": "blockly-scripts clean",
-    "lint": "blockly-scripts lint --fix",
+    "lint": "eslint . --fix",
```
Comment (ewpatton) [inline package.json:9]: Is there a reason you switched to eslint versus using Blockly's rules?

**Review by ewpatton — APPROVED** [review]

**Pull request merged**